ui/client/datasets/DataTransformation/DataTransformation.js

```diff
@@ -692,6 +683,11 @@ const DataTransformation = ({
     setJobsConfig(transformations);
     // The actual jump to next step happens in the useEffect at the top of this function
     // after the jobs have completed in RunTransformation
+
+    // if there are no transformations, toggle this to trigger our 'handleNext' useEffect at top
```

Comment:
We need to do this now as we are no longer forcing the user to complete at least one transformation